hdfs-scheduler/src/main/java/org/apache/mesos/hdfs/scheduler/Reconciler.java

```diff
@@ -117,4 +124,42 @@ private void explicitlyReconcileTasks(SchedulerDriver driver) {
 
     driver.reconcileTasks(tasks);
   }
+
+  private class ReconcileThread extends Thread {
+    private static final int MILLIS_FROM_SECONDS = 1000;
```

Comment:
Let's work to make the style more consistent and if there is some disagreement, then we can gladly discuss. Yes, I understand there is a certain amount of tech debt there that is not your fault. Discussing and making simple style changes should not be too tedious.